typespec-extension/src/emitter.ts

```diff
@@ -59,7 +59,7 @@ const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
 };
 
 export const $lib = createTypeSpecLibrary({
-  name: "JavaEmitter",
+  name: "@azure-tools/typespec-java",
```

Comment:
What is this change's impact?

Reply:
I've sent an email about this "FW: Steps to Add Nightly Pipeline Testing Latest Tsp Releases".

Change in (dev) compiler make this name matters now.

Reply:
Though nightly still fails (at a later place...)